src/usdb_syncer/path_template.py

```diff
@@ -163,13 +163,16 @@ def evaluate(self, song: UsdbSong) -> str:
             case PathTemplatePlaceholder.TITLE:
                 return song.title
             case PathTemplatePlaceholder.GENRE:
-                return song.genre
+                firstGenre, _, _ = song.genre.partition(',')
```

Comment:
Why not use `return song.genre.split(',')[0]?

Reply:
Couple of reasons:
1. No out of bounds check needed cause index 0 could be not set in the case of split with an empty string leading to IndexError
2. Predictable output. Partition always returns a tuple of 3 values: <Item before the comma> | <the comma> | <rest of string>. As mentioned in the PR we don't need the other 2 values
3. Performance wise; split enumerates the whole array whereas partition stops at the first occurence
4. Readability

Reply:
If this is alright i'll resolve the convo :)